### ISSUE TITLE
[Feature Request] complete interaction with Vimium C if its installed, or just add more keybindings for faster workflow

### ISSUE THREAD

**Issue by fislysandi** [mbnuqw/sidebery#869]
## Description
basically detect if vimium c is installed and extend the functionality of sideberry if its installed,
looking at what mode you are currently in ie.normal mode,insert mode. these keymaps below are possible.
basically insert,visual mode disables these keybindings

==========================================================================

or in general what keybindings i would like to see added and allow them to be re mappable

w jump to the next tab tree indent
Shift + w jump to the previous Tab tree indent

e jump to the next tab tree but expand the tab and move inside of it
shift + e do the opposite of e but still be inside of the tab

a motion that allows you to select the insides of an active tab tree (you can input a2k to select 2 indented tabtrees that are attached above and a2j to select indented tabtrees attached below ,like this motion a2>2 move 2 tab indents 2 tabs forward or a2>ww move 2 tab indents to the next tab tree, etc. you can go nutty with this )

aa select active tab tree 

alt + v to enter tab visual mode and escape to close out of it (maybe there needs to be a keybinding to deselect tabs or re select them)

vimium C already has a keybinds to move tabs that is >> <<

i could go on and on what couldve been done with this interaction but i dont want to
==================================================================================
basically this is an idea that came into my head and this will likely will never be made but still its a cool idea. 

the next feature request i would post most likely you will be intrested in doing and that is 
https://github.com/mbnuqw/sidebery/issues/868

**Comment by semi710** [comment]
Is this feature is in consideration?

Or either we can have separate set of key mapping that can be mapped only when sidebar is visible?
